product_docs/docs/pgd/3.7/harp/01_release_notes/harp2.4.1_rel_notes.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: "Version 2.4.1"
+---
+
```

Comment:
Do we need to add here "Release date: ...."

Reply:
Fixed.